### DISCUSSION TITLE
footer section not found

### DISCUSSION THREAD

**Discussion by Pushpa472** [community/community#140976] [Programming Help]
### Body

@davidstaheli @mgriffin   I search it but still not found...
![image](https://github.com/user-attachments/assets/0dc4335b-d2b2-4cd5-8798-c052c6f249fb)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by RexActor** [comment]
Looks like you don't have branch named `gh-pages`

To check available branches on your repo, use `git branch` command and it will list all available branches

and if you don't have that branch name in list, you can create new branch by using command `git branch -b <branch name>`

**Comment by Pushpa472** [comment]
@RexActor I fork all repo , i click on gh-pages open in codespaces it not show and also do git checkout gh-pages even in main repo not show and i create my feature branch footer-section....  
![image](https://github.com/user-attachments/assets/85cd81ec-d349-47f2-ae59-b46713f6dd45)

**Comment by RexActor** [comment]
as far as I'm aware, when you fork repo, it's not copying branches

`Forking a repository will copy data such as files and code but Issues, branches, pull requests and other features won't join the party. Instead, your fork will start the same way as a newly created repository so you can begin work on it as a fresh project.`

[See it here ](https://github.com/orgs/community/discussions/35849)

**Comment by Himani0002** [comment]
I am a full stack development student and BCA.

Tell me about any internship or anything you might know.

please

**Comment by dalvarez2596** [comment]
@Pushpa472  did you try `git fetch` ?
This is the git fetch info :
https://git-scm.com/docs/git-fetch
